### ISSUE TITLE
mypy --fast-parser error even when dependency is satisfied?

### ISSUE THREAD

**Issue by Integralist** [python/typed_ast#35]
I have the following `Dockerfile`:

```Dockerfile
FROM python:3.6.1

WORKDIR /tmp

RUN apt-get update -y
RUN apt-get install -y git
RUN git clone https://github.com/vim/vim.git && cd vim && ./configure && apt install ncurses-dev && make && make install
RUN vim --version | head -n 1

ADD ./requirements.txt /app/requirements.txt
RUN pip install -r /app/requirements.txt

COPY .vim /root/.vim
COPY .vimrc /root/.vimrc

WORKDIR /app
```

Here is my requirements file:

```text
flake8==3.2.1
flake8-deprecated==1.1
flake8-docstrings==1.0.3
flake8-mock==0.3
flake8-quotes==0.9.0
mypy==0.470
pep8-naming==0.4.1
pylint==1.6.4
pytest==3.0.5
```

When I open a Python file inside the container I see the message:

```
You must install the typed_ast package before you can run mypy with `--fast-parser`.
You can do this with `python3 -m pip install typed-ast`.
```

But if I install that as suggested I see:

```
Requirement already satisfied: typed-ast in /usr/local/lib/python3.6/site-packages
```

**Comment by kirbyfan64** [comment]
https://github.com/python/mypy/issues/2862

**TL;DR:** Latest `typed-ast` requires mypy 0.501.

**Comment by Integralist** [comment]
Cool, thanks 👍
